### PR TITLE
Feature/login UI

### PR DESCRIPTION
**Pull request by princekumarLnG**
made layout for login page and added file for making layout for other pages.
following snip is of page after using layout.
image

### CODE REVIEW

File: Pspcl/Pspcl.Web/Views/Account/Login.cshtml [hunk 1]
```diff
@@ -1,90 +1,58 @@
...
-</head>
-<body>
+@{
+    ViewData["Title"] = "Login";
+}
+
+@section styles {
     <style>
-        #electricMeter{
+        #electricMeter {
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Views/Account/Login.cshtml]: Move these styles into site's css file

File: Pspcl/Pspcl.Web/Views/Account/Login.cshtml [hunk 2]
```diff
@@ -1,90 +1,58 @@
...
             height: 400px;
-            width: 400px;            
+            width: 400px;
         }
-        .bg{
-            background-color:#e3e4e5;
+
+        .bg {
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Views/Account/Login.cshtml]: Avoid adding styles inline. It should be in a css file

File: Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+
+
+    <footer class="border-top footer text-muted mt-5">
+        <div class="container">
+            <p><center>Powered By LnGConsultancy</center></p>
+        </div>
+    </footer>
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js" integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN" crossorigin="anonymous"></script>
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml]: 2 bootstrap jquery files reference given. The order of files is also wrong. Jquery library should come first, then bootstrap. You can remove bootstrap bundle JS and use bootstrap.min.

Comment (princekumarLnG) [reply Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml]: Resolved now. following is the order now:  **jquery-3.5.1.min.js,  popper.min.js,  bootstrap.min.js**

File: Pspcl/Pspcl.Web/Views/Account/Login.cshtml (hunk 1 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.Web/Views/Account/Login.cshtml]: ok👍. understood

File: Pspcl/Pspcl.Web/Views/Account/Login.cshtml (hunk 2 above)
Comment (princekumarLnG) [reply Pspcl/Pspcl.Web/Views/Account/Login.cshtml]: ok👍. understood. will keep in mind.

File: Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml (hunk 3 above)
Comment (sandeepkumarlng) [reply Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml]: popper min should be after bootstrap. Do confirm from your research also.

Comment (princekumarLnG) [reply Pspcl/Pspcl.Web/Views/Shared/NoSideBarLayout.cshtml]: i researched , and used chatGpt also, it said the same order

**Review by sandeepkumarlng — APPROVED** [review]

**Pull request merged**